pytket/tests/classical_test.py

```diff
@@ -481,6 +482,109 @@ def test_add_wasm_to_reg() -> None:
     assert c.depth() == 4
 
 
+def test_wasm_argtypes() -> None:
+    w = wasm.WasmFileHandler("testfile.wasm")
+
+    with open("testfile.wasm", "rb") as f:
+        bytecode = f.read()
+    assert w.bytecode() == bytecode
+
+    c = Circuit(0, 6)
+
+    c.add_wasm("add_one", w, [1], [1], [Bit(0), Bit(1)])
+
+    assert c.depth() == 1
+
+    cmd = c.get_commands()[0]
+
+    assert isinstance(cmd.args[0], unit_id.Bit)
+    assert isinstance(cmd.args[1], unit_id.Bit)
+    assert isinstance(cmd.args[2], unit_id.WasmState)
```

Comment:
Can you also remove the workaround in `pytket/pytket/circuit/decompose_classical.py` line 513? Shouldn't be needed now.